edenai_apis/apis/microsoft/microsoft_text_api.py

```
@@ -66,6 +66,38 @@ def text__moderation(
             original_response=data, standardized_response=standardized_response
         )
 
+    async def text__amoderation(
+        self, language: str, text: str, model: Optional[str] = None, **kwargs
```

Ruff (0.12.2)

Unused method argument: `model`

(ARG002)